src/openrct2/ride/Vehicle.cpp

```diff
@@ -1830,6 +1830,8 @@ static void vehicle_update_measurements(rct_vehicle* vehicle)
         // Set tile_element to first element. Since elements aren't always ordered by base height,
         // we must start at the first element and iterate through each tile element.
         auto tile_element = map_get_first_element_at(x / 32, y / 32);
+        if (tile_element == nullptr)
+            return;
```

Comment:
This is okay but it is a tad messy it should really continue onto the code below like when surfaceelment is null. It will never happen though since map_get_surface_element_at already calls get_first_element_at with these coordinates.

Reply:
Ok, I'm assuming nothing needs to be done here, for now.

Reply:
Yeah it's fine